mongodbatlas/resource_mongodbatlas_project.go

```diff
@@ -292,14 +295,62 @@ func resourceMongoDBAtlasProjectDelete(ctx context.Context, d *schema.ResourceDa
 	conn := meta.(*MongoDBClient).Atlas
 	projectID := d.Id()
 
-	_, err := conn.Projects.Delete(ctx, projectID)
+	stateConf := &resource.StateChangeConf{
+		Pending:    []string{"DELETING", "PENDING", "REPEATING"},
+		Target:     []string{"IDLE"},
+		Refresh:    resourceProjectClustersDeleteRefreshFunc(ctx, projectID, conn),
+		Timeout:    3 * time.Hour,
```

Comment:
[q] How the value of the timeout has been decided? 3 hours seems a lot

Reply:
The timeout time was taken from the [&resource.StateChangeConf in resourceMongoDBAtlasAdvancedClusterDelete](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/a5132e87556a4b08ef2d77f44b38dbcf3bd316ea/mongodbatlas/resource_mongodbatlas_advanced_cluster.go#L643). I agree though, it does seem too long.

Reply:
It's been so long I don't remember how 3 hours was arrived at or if it was even discussed.  It is a long time though and the majority of cluster changes, even very large ones, won't take that long.   @andreaangiolillo any suggestions from your experience?   I don't think I've seen a deletion take more than 30 mins and that's still pretty generous.

Reply:
30 mins sounds good to me 👍

Reply:
:+1: fixed